aerogear/mobile-security-service-operator
# feat(Allow usage of apply command for App CR) : Rebind without the need to delete and create again

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9268

## What
- Ensure that always will have just 1 ConfigMap created with the latest name of APP CR
- Allow usage of the apply command 
- Fix debug after all changes.
- Fix/improve tests after rebase
- Use the model from MSS instead of duplicate it.
- Upgrade operator-sdk from 0.7.0 to the latest version available 0.8.1 

## Verification Steps
Use the image:  docker.io/cmacedo/mobile-security-service-operator:AEROGEAR-9268

1. Install the oper
2. Create the app in the specific namespace
3. Check that a configMap was created for this app
4. Update the name into the Example APP CR 
5. Run the command `make create-app` again
6. It should have just one confgMap with the new name and not 2. 

## Checklist:


## Progress


## Additional Notes

<img width="1578" alt="Screenshot 2019-05-28 at 10 45 44" src="https://user-images.githubusercontent.com/7708031/58468615-ca5f3e00-8135-11e9-822b-59e53e550b53.png">
